install-deps.sh

```diff
@@ -306,9 +306,11 @@ else
     [ $WITH_ZBD ] && with_zbd=true || with_zbd=false
     [ $WITH_PMEM ] && with_pmem=true || with_pmem=false
     source /etc/os-release
+    pip install jsondiff
     case "$ID" in
     debian|ubuntu|devuan|elementary)
         echo "Using apt-get to install dependencies"
+        $SUDO apt-get install -y jsonnet
```

Comment:
Also this needs to be added to the `debian/control` file instead.

Reply:
Done